examples/202-NN_examples.ipynb

```diff
@@ -42,7 +42,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    std = StandardScalerTransform(in_column=["target"] + lag_columns)
</pre><p>May be it will make it more readable to separate features and scallers </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/268/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>